app/models/label.rb

```diff
@@ -37,13 +37,24 @@ class Label < ApplicationRecord
 private
 
   def update_level_if_label_defined
-    return unless Level.exists?(key: key)
+    return unless scope == 'private'
     account.reload.update_level
+    send_document_review_notification if key == 'document'
+  end
+
+  def send_document_review_notification
+    if value == 'verified'
+      ProfileReviewMailer.approved(account).deliver_now
+      EventAPI.notify('system.document.verified', account_uid: account.uid)
```

Comment:
add `TODO: Need to update specs/event_api.md` if you don't add it in current request. Specs are important for users of Events API